calico/tests/common.py

```diff
@@ -27,7 +27,6 @@
     "calico.felix.active.local_endpoints",
     "calico.felix.active.local_policies",
     "calico.felix.active.local_selectors",
-    "calico.felix.active.local_tags",
```

Comment:
Could we update the description of this metric to let the customers know it is not available anymore with versions higher than 2.24?

Reply:
There is a deprecation notice in the description already: https://github.com/DataDog/integrations-core/blob/625e46850795b94f2a5582b5ab6736e81fdaf0e6/calico/metadata.csv#L5

Reply:
Ah. Since we are removing it from the tests I thought it was dropped in 2.24, which is not exactly the same as deprecated

Reply:
They were dropped in 2.23, I guess the word _deprecated_ may be a bit misleading here. I'll try to clarify that.